src/main/java/com/google/sps/image/ImageSelection.java

```diff
@@ -90,18 +98,53 @@ private static float findBestImage(ImageScorer imageScorer, StringBuilder bestIm
     return bestImageScore;
   }
 
+  private List<String> addBackups(List<String> images, int noBackups) {
```

Comment:
I guess noBackups means number of backups but is a poor choice for a variable name since can cause confusion with "actually no backups"

Reply:
I really thought that "no" comes "number of" :).

Reply:
Done.